dtschema/dtb.py

```diff
@@ -262,7 +262,6 @@ def fdt_scan_node(fdt, nodename, offset):
     'msi-ranges': '#interrupt-cells',
     'gpio-ranges': 3,
 
-    'nvmem-cells': None,
```

Comment:
Independently, Rafał Miłecki [reported a bug](https://lore.kernel.org/all/bdf7751b-0421-485d-8382-26c084f09d7d@gmail.com/) regarding nvmem-cells and I narrowed it to this line. Later I found this pull request.

I think this part is needed along with last hunk...